seed.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Welcome to the seed file! This seed file uses a newer language feature called...
+ *
+ *                  -=-= ASYNC...AWAIT -=-=
+ *
+ * Async-await is a joy to use! Read more about it in the MDN docs:
+ *
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
+ *
+ * Now that you've got the main idea, check it out in practice below!
+ */
+const db = require('./server/db')
+const {User} = require('./server/db/models')
+
+async function seed () {
+  await db.sync({force: true})
```

Comment:
Maybe log `connecting to db` and `connected` before and after this line? Not sure if that's too "cute" / superfluous / noisy. Could help drive home what's happening though.